CHANGELOG.md

```diff
@@ -6,6 +6,7 @@
 - Several file extensions are interpreted as LaTeX file ([#81](https://github.com/area/language-latex/pull/81), [#82](https://github.com/area/language-latex/pull/82), and
 [#126](https://github.com/area/language-latex/pull/126))
 - Support for highlighting long lines ([#131](https://github.com/area/language-latex/pull/131))
+- Added spec
```

Comment:
Move this line on top of the file. Your PR will be published in a future release, not in v1.1.0, which is already out. Also please use imperative mood instead of past tense.